pyproject.toml

```diff
@@ -43,7 +43,10 @@ ua-parser = { version = ">=0.10", optional = true }
 black = "*"
 coverage = "*"
 flake8 = "*"
+furo = "*"
 isort = "*"
+sphinx = "*"
+sphinxcontrib-spelling = "*"
```

Comment:
I wonder if `sphinxcontrib-spelling` can be a soft requirement, since it requires `pychant`, which requires `enchant`, which requires being set-up correctly with...? 🤔

The Django docs have: 

```
if "spelling" in sys.argv:
    extensions.append("sphinxcontrib.spelling")
```

… for this.

Reply:
I made it optional.